exec/exec_test.go

```diff
@@ -388,7 +396,7 @@ func BenchmarkModules(b *testing.B) {
 			if err != nil {
 				b.Fatal(err)
 			}
-			runTest(path, testCases, b)
+			runTest(path, testCases, b, false)
```

Comment:
I haven't checked yet in the later PRs but I assume we'll have the option to benchmark with native=true :)

Reply:
For now, we run both the tests twice (once with native and once with non-native). If you are on a platform with no native support, both tests are identical. I can change the behaviour if you would rather only run them once using a build tag.

Reply:
the current behaviour is fine for tests.
for benchmarks, I guess it would be great to have a way to show the AOT improvements.
(but this can wait for when the PRs w/ the actual impl is there.)

Reply:
SGTM :) I plan on the following (not in PRs yet, but sometime in the future):

- Exposing an API on the VM/func so you can see what sequences were compiled to native code
- Fixing the benchmarks, making a few common cases (like hashing)
 - Writing some docs on the status of the native backend & how to enable it.